.github/workflows/build-and-test.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - 7onn/update-metabase-dashboard-url
```

Comment:
```suggestion
```

Reply:
mainly just need to do this before merging the PR in

Reply:
done by: https://github.com/datahub-project/datahub/pull/9631/commits/d9022bc4938e53471161457a33f8e4e03463c272